### ISSUE TITLE
Add a new phase popup changes

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#2108]
We have to modify the catalog of the phases we can add to a project. Right now we should

- [ ] Update the title
- [ ] Remove the help text at the footer
- [ ] Remove (solution) projects from the catalog - app, website

We're working on the updated catalog that should allow the managers/copilots to add phases, based off of https://docs.google.com/spreadsheets/d/1TACVoX-Q7HT8f4DPGCYojltmgszJEB64H3GbNZPARzQ/edit?usp=sharing

The catalog would offer different iterations and addons, so we have to provide final design.

**Comment by vikasrohit** [comment]
yes, final designs would be helpful. Right now we don't have enough information to proceed on this. We need a simple list of products with its attributes to be displayed on the UI to start working on it. It is very difficult for us to identify this information from that master sheet.

**Comment by rmjmccormick** [comment]
@vikasrohit While we await the final designs, please proceed with the following changes which represent the first three steps of @vic-topcoder message above

![image](https://user-images.githubusercontent.com/39598852/42157070-05d7049c-7ded-11e8-870e-881d09414951.png)

**Comment by vikasrohit** [comment]
First two items are done. For the third item we have to identify the catalog first and should handle it separately.
